Traceback (most recent call last):
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/joao/opt/anaconda3/envs/teaching/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
:tags: ["hide-input"]

import numpy as np
import matplotlib.pyplot as plt

Nx=100
X=np.linspace(0,1,Nx)

fig, ax0 = plt.subplots()
ax0.set_xlim(( 0, 1))
ax0.set_ylim((-0.1, 1.1))

line1, = ax0.plot(X, topHat(X), lw = 2, color = "b",  label='f')

ax0.set_xlabel('x')
ax0.set_ylabel('f')
ax0.set_title("Initial Condition")
ax0.legend()
------------------

[0;36m  Cell [0;32mIn[2], line 1[0;36m[0m
[0;31m    :tags: ["hide-input"][0m
[0m    ^[0m
[0;31mSyntaxError[0m[0;31m:[0m invalid syntax

SyntaxError: invalid syntax (2128274755.py, line 1)

